src/js/components/Layer/__tests__/__snapshots__/Layer-test.js.snap

```diff
@@ -2017,8 +2017,8 @@ exports[`Layer invokes onEsc when modal={false} 1`] = `
   display: flex;
   box-sizing: border-box;
   max-width: 100%;
-  margin-left: 12px;
-  margin-right: 12px;
+  margin-left: 16px;
```

Comment:
Changes only due to jsdom, does not appear in browser. Correct value is re-assigned below